source/geod24/LocalRest.d

```diff
@@ -527,6 +527,7 @@ public final class RemoteAPI (API) : API
         Duration timeout = Duration.init)
     {
         auto childTid = C.spawn(&spawned!(Impl), args);
+        childTid.setTimeout(timeout);
```

Comment:
If the timeout is set on the Tid directly, there is no need to pass it to `RemoteAPI`'s constructor, is there ?